Examples/ObjectiveC/FeatureSelectionExample.m

```diff
@@ -85,13 +82,10 @@ - (void)changeOpacityBasedOn:(NSString*)name {
     
     // Check if a state was selected, then change the opacity of the states that were not selected.
     if (name.length > 0) {
-        layer.fillOpacity = [MGLStyleValue valueWithInterpolationMode:MGLInterpolationModeCategorical
-                sourceStops:@{ name: [MGLStyleValue valueWithRawValue:@1] }
-                attributeName:@"name"
-                options:@{ MGLStyleFunctionOptionDefaultValue: [MGLStyleValue valueWithRawValue:@0] }];
+        layer.fillOpacity = [NSExpression expressionWithFormat:@"MGL_MATCH(name, %@, 1, 0)", name];
```

Comment:
I think this would be easier to understand:

```objc
layer.fillOpacity = [NSExpression expressionWithFormat:@"TERNARY(name = %@, 1, 0)", name];
```